server/restivus-swagger.js

```diff
@@ -5,17 +5,21 @@ const URL = new ParsedURL(Meteor.absoluteUrl());
 
 // Add swagger route and generate valid swagger.json
 Restivus.prototype.addSwagger = function(path) {
```

Comment:
What path is this? Consider naming it `swaggerPath`, so that it is easy to understand in later code.